validation.php

```diff
@@ -0,0 +1,29 @@
+<?php
+    /**
+    *  Used for check email format valid or not.
+    */
+    class validation {
+        /**
```

Comment:
Make this change wherever applicable.
```suggestion
        
        /**
```